lib/internal/priority_queue.js

```diff
@@ -1,14 +1,6 @@
 'use strict';
 
-const {
-  Array,
-  Symbol,
-} = primordials;
-
-const kCompare = Symbol('compare');
-const kHeap = Symbol('heap');
-const kSetPosition = Symbol('setPosition');
-const kSize = Symbol('size');
+const { Array } = primordials;
```

Comment:
To minimize the diff
```suggestion
const {
  Array,
} = primordials;
```